### PR TITLE
Fix documentation errors regarding UTF8

### PR DESCRIPTION
**Pull request by mmatera**
Well, 
I was able to fix the utf8 errors using xelatex instead of pdflatex, and the issue of the empty tests. There are still errors coming from the asy interface for graphics generation. I am not an expert on that package, so I will need some help from here. Otherwise, we could simply remove the tests that generate problematic graphics.

### CODE REVIEW

File: mathics/doc/tex/Makefile [hunk 1]
```diff
@@ -1,10 +1,11 @@
-TEX = pdflatex
+#TEX = pdflatex
```
Comment (GarkGarcia) [inline mathics/doc/tex/Makefile:1]: We should probably remove this comment. It may  cause confusion in the future.

Comment (GarkGarcia) [reply mathics/doc/tex/Makefile:1]: This is the only issue I found in this commit.

Comment (mmatera) [reply mathics/doc/tex/Makefile]: I have removed it. Shall I make a rebase first?

Comment (GarkGarcia) [reply mathics/doc/tex/Makefile]: > It's not necessary. There are no conflicts in your branch 😁️

**Pull request merged**